# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/android/AccessibleWrap.cpp
# Commit: 7488e7ba80c5
# Full Hash: 7488e7ba80c5973f1444fa7502c84a95e90a7f2c
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 3: Directly handle scrolling start events on Android instead of converting them to virtual cursor change events. r=eeejay
#   
#   Depends on D192643
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192644
# ==============================================================================

diff -r 3d0747ce0174 -r 7488e7ba80c5 accessible/android/AccessibleWrap.cpp
--- a/accessible/android/AccessibleWrap.cpp	Tue Nov 28 22:03:19 2023 +0000
+++ b/accessible/android/AccessibleWrap.cpp	Tue Nov 28 22:03:20 2023 +0000
@@ -61,19 +61,6 @@
 nsresult AccessibleWrap::HandleAccEvent(AccEvent* aEvent) {
   auto accessible = static_cast<AccessibleWrap*>(aEvent->GetAccessible());
   NS_ENSURE_TRUE(accessible, NS_ERROR_FAILURE);
-  DocAccessibleWrap* doc =
-      static_cast<DocAccessibleWrap*>(accessible->Document());
-  if (doc) {
-    switch (aEvent->GetEventType()) {
-      case nsIAccessibleEvent::EVENT_SCROLLING_START: {
-        accessible->PivotTo(
-            java::SessionAccessibility::HTML_GRANULARITY_DEFAULT, true, true);
-        break;
-      }
-      default:
-        break;
-    }
-  }
 
   nsresult rv = LocalAccessible::HandleAccEvent(aEvent);
   NS_ENSURE_SUCCESS(rv, rv);